Clients/Xamarin.Interactive.Client.Desktop/Workbook/Views/XcbWorkbookPageView.cs

```diff
@@ -166,7 +167,9 @@ protected override void OnCompilationWorkspaceAvailable ()
                 getSourceTextByModelId);
         }
 
-        public override Task LoadWorkbookDependencyAsync (string dependency)
+        public override Task LoadWorkbookDependencyAsync (
+            string dependency,
+            CancellationToken cancellationToken)
```

Comment:
I thought we don't add CT params when cancellation isn't possible?

Reply:
This is an `override` and it implements `IEvaluationService. LoadWorkbookDependencyAsync`.

Reply:
This will probably fall out as well once XCB/WPVM is gone anyway.